### PR TITLE
Create mergesortedlists lab 

### PR DESCRIPTION
**Pull request by jlge**
Created C lab with a C autograder.

Included the requested autograde.tar, autograde-Makefile, reference student submission file, and uncompressed autograde.tar directory.

If necessary, I can add the src directory including handout files for students as well.

### CODE REVIEW

File: c-mergesortedlists/mergesortedlists-autograder/driver.c [hunk 1]
```diff
@@ -0,0 +1,212 @@
...
+
+  for (int i = 0; i < 4; i++) {
+    score += testResults[i];
+    printf("Test %d passed: %s\n", i+1, testResults[i] ? "true" : "false");
+  }
+
+  printf("{ \"scores\": { \"problem1\": %d } }", score);
+
+  return 0;
+}
```
Comment (20wildmanj) [inline c-mergesortedlists/mergesortedlists-autograder/driver.c]: Nit: add newline to end of file

File: c-mergesortedlists/mergesortedlists-autograder/mergesortedlists.h [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+ * @file mergesortedlists.h
+ * @brief Header file for merging two sorted list function implementation.
+ */
+
+typedef struct list_ele {
+    int val;
+    struct list_ele *next;
+} list_t;
+
+list_t *mergeTwoSortedLists(list_t *list1, list_t *list2);
```
Comment (20wildmanj) [inline c-mergesortedlists/mergesortedlists-autograder/mergesortedlists.h]: Nit: add newline to end of file

File: c-mergesortedlists/mergesortedlists.h [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+ * @file mergesortedlists.h
+ * @brief Header file for merging two sorted list function implementation.
+ */
+
+typedef struct list_ele {
+    int val;
+    struct list_ele *next;
+} list_t;
+
+list_t *mergeTwoSortedLists(list_t *list1, list_t *list2);
```
Comment (20wildmanj) [inline c-mergesortedlists/mergesortedlists.h]: Nit: add newline to end of file

**Review by 20wildmanj — APPROVED** [review]
Was able to run the autograder successfully on Nightly.

Only have a nit about newlines, otherwise LGTM

**Comment by jlge** [comment]
Added newlines where necessary!

**Pull request merged**